packages/react-components/src/helpers/options-for-select.js

```diff
@@ -1,43 +1,3 @@
-export const maritalStatuses = [
```

Comment:
All of these are unused in Component Library and were copy and pasted from vets-website. Removing since it is unused code.